specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2019-09-01/providers.json

```diff
@@ -108,6 +108,10 @@
           "x-ms-client-flatten": true,
           "x-ms-client-name": "OperationProperties",
           "$ref": "#/definitions/OperationProperties"
+        },
+        "isDataAction": {
```

Comment:
Did the service actually return this for 2019-09-01? If it's new, it should only be in 2020-04-01. GA'd libraries shouldn't be added to.

Reply:
And, is this absent if not defined, or does the service return an explicit `null`? It's important to mark anything that would have an explicit `null` value in the JSON response as `x-nullable: true`.

Reply:
Yes. The object missing properties error is reported in 2019-09-01 version. This property value is either true or false in the response. Following is a piece of response in 2019-09-01 with this property.

 **_{
          "name": "Microsoft.KeyVault/operations/read",
          "display": {
            "provider": "",
            "resource": "",
            "operation": "",
            "description": ""
          },
          "isDataAction": false
        }
}_**